pkg/operators/machine-api-operator.go

```diff
@@ -14,7 +14,7 @@ var (
 	maoManagedDeployment = "machine-api-controllers"
 )
 
-var _ = Describe("[Feature:Operators] Machine API operator deployment should", func() {
+var _ = Describe("[Feature:Operators] Machine API operator deployment should [Serial]", func() {
```

Comment:
is this something ginkgo just understand? can you reference the docs?

Reply:
@enxebre @JoelSpeed You are right, not at the moment. Here is a separate PR for serial support: https://github.com/openshift/cluster-api-actuator-pkg/pull/186

Reply:
these labels are inspired by the kubernetes community docs[0]. ginkgo doesn't know them, but i have started to clean them up in the autoscaler tests as i think it helps people to understand how to operate the tests.

additionally, as demonstrated in Danil's pr, they can be used to filter the tests.

Reply:
Out of interest, do you have any estimate how much time this might add to each test run by making these serial? Do you think there are any tests within here that can still run in parallel? We could maybe divide them into parallel and serial sets